cli/src/main.rs

```diff
@@ -105,6 +105,10 @@ struct CliArgs {
     #[clap(long = "certs")]
     cert_chain: bool,
 
+    /// Do not enforce trust lists. (CAWG-only at the moment.)
+    #[clap(long)]
+    disable_cawg_trust: bool,
```

Comment:
This does not need to be a command line option, it belongs only in settings.

Reply:
+1, I also think this should be "only" a setting. You can also use settings with the cli anyway.